test/onnx/exporter/test_capture_strategies.py

```diff
@@ -15,6 +15,7 @@ class ExportStrategiesTest(common_utils.TestCase):
         [
             _capture_strategies.TorchExportStrategy,
             _capture_strategies.TorchExportNonStrictStrategy,
+            _capture_strategies.TorchExportDraftExportStrategy,
```

Comment:
We should start the habit of documenting significant changes. This one is significant. It should be mentioned in torch.onnx.export.

Reply:
How much info do you think we should share in the docstring? In some ways this is an implementation detail now that I think of it.

Reply:
It should be written somewhere. If it needs to be mentioned somewhere else, a link to the first can be added. I like it is mentioned in the docstring because it is closer to the code so it is easier to remember to change it. It does not have to thorough but users must have enough information to understand the report when the conversion fails. Strict, non strict, jit, every of them can be described with a link to the page in pytorch documentation describing them.

Reply:
Added descriptions for the strategies in torch.onnx.export documentation